www/Makefile

```diff
@@ -20,15 +20,12 @@ clean:
 	-rm -rf _build/*
 
 upload: html
-	cd _build/html && \
-	    touch .nojekyll && \
-	    echo www.numpy.org > CNAME && \
-	    rm -rf .git && \
-	    git init && \
-	    git remote add target git@github.com:numpy/numpy.github.com.git && \
-	    git add -A && \
-	    git commit -m "Rebuild site" -a && \
-	    git push -f target master:master
+	git clone git@github.com:numpy/numpy.github.com built_doc
```

Comment:
This is going to be extremely slow right, if `numpy.github.com` grows really large? I'm not sure I understand the logic of recloning the generated docs here. PR title sounds useful, but don't quite see how this change achieves it.

Reply:
I don't see how to resolve the conflict between the desire to have [fewer repos for the documentation](https://github.com/numpy/numpy/pull/13921#issuecomment-508881361) and on the other hand not clone it.

Reply:
See my other comment on this PR: one for docs, one for numpy.org

Also, it would be good to summarize how you plan to implement this before doing everything, that's way easier for both of us. I'm not trying to be super picky, but as I'll probably be the main maintainer of this long-term _and_ have plans for numpy.org (as you know), this setup is actually important.